ansible/library/killbill

```diff
@@ -17,12 +17,18 @@ unless data['kpm_path'].nil?
 end
 require 'kpm'
 
+
+
 # Temporary -- https://github.com/killbill/killbill-cloud/issues/91
 log = StringIO.new
 logger = Logger.new(log)
 logger.level = Logger::INFO
 
 kpm_yml = data['kpm_yml'].is_a?(String) ? YAML.load_file(data['kpm_yml']).to_hash : data['kpm_yml'].to_hash
+
+tmp = kpm_yml["killbill"]["nexus"]
```

Comment:
I'm wondering if we shouldn't change `base_artifact.rb` instead: https://github.com/killbill/killbill-cloud/blob/82e6d86262865526b426e58942661e5ddc9dd343/kpm/lib/kpm/base_artifact.rb#L46

since we expect string keys for everything else in the config.

Something like:

```
def nexus_remote(overrides={}, ssl_verify=true, logger=nil)
  # overrides typically comes from the kpm.yml where we expect keys as String
  overrides_sym = (overrides || {}).each_with_object({}) {|(k,v), h| h[k.to_sym] = v}
  nexus_config = nexus_defaults.merge(overrides_sym)
  nexus_remote ||= KPM::NexusFacade::RemoteFactory.create(nexus_config, ssl_verify, logger)
end
```

Thoughts?